src/content/blog/2024-02-21-eslint-2023-year-review.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: ESLint's 2023 year in review
+teaser: "2023 saw the much-anticipated release of our new configuration system and more."
+tags:
+  - Updates
+  - Features
+  - Releases
+authors:
+  - nzakas
+categories:
+  - Storytime
+---
+
+The plan for 2023 centered around our first major release [since 2021](https://eslint.org/blog/2021/10/eslint-v8.0.0-released/), ESLint v9.0.0. Most of the year was spent preparing for the release, building out the new configuration system (flat config) and communicating the impact to the ecosystem throughout the year.
+
+## Release of ESLint v9.0.0 and the new configuration system
+
+While there are [a lot of changes in v9.0.0](https://eslint.org/blog/2023/11/whats-coming-in-eslint-9.0.0/), the biggest change is making flat config the default configuration system. The old (eslintrc) configuration is still available via an environment variable, and we published the [rollout plan](https://eslint.org/blog/2023/10/flat-config-rollout-plans/) outlining how the transition will occur from v8.x through to v10.x.
+
+Just before the end of the year, we [released ESLint v9.0.0-alpha.0](https://eslint.org/blog/2023/12/eslint-v9.0.0-alpha.0-released/). We anticipate a final v9.0.0 release in the spring of 2024.
+
+## Other changes
+
+* [Deprecation of formatting rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules/) - in a significant shift from ESLint's roots, we decided to officially deprecate formatting rules
+* [Changes to rule API](https://eslint.org/blog/2023/09/preparing-custom-rules-eslint-v9/) - for those who write their own rules, we announced important changes to the rule API that will prepare ESLint for our next step: [language plugins](https://github.com/eslint/rfcs/blob/main/designs/2022-languages/README.md).
+
+## Updates to support the latest ECMAScript features
+
+The team spends much of the year anticipating new ECMAScript features and updating ESLint accordingly. 2023 was an outlier in that most of the changes weren't syntax-related, and so there wasn't a lot of additional work. The most significant syntax change, [hashbang comments](https://github.com/tc39/proposal-hashbang), is something that ESLint already handled to support Node.js.
+
+## Income review
+
+During 2023, ESLint received $179,601.53 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
+
+| **Source** | **Amount** |
+|-----------|-------------|
+| Open Collective | $117,336.30 |
+| Tidelift | $30,077.84 |
+| GitHub Sponsors | $25,190.89 |
```

Comment:
Haven't we connected Github sponsors with opencollective? [Ref contribution](https://opencollective.com/eslint/contributions/713677) So, should we add this to the total? Isn't this already included in Open Collective balance?

<img width="255" alt="Screenshot 2024-02-21 at 3 20 49 AM" src="https://github.com/eslint/eslint.org/assets/46647141/78e6d1ac-6502-4e12-a8d2-dc1dfd475767">

Reply:
Yes, GitHub Sponsors donations get deposited to Open Collective, but I break it out here to better show where the donations were initiated from.

Reply:
make sense 👍🏻